ambry-store/src/main/java/com/github/ambry/store/BlobStoreCompactor.java

```diff
@@ -1393,6 +1393,17 @@ private boolean isDeleteTombstoneRemovable(IndexEntry deleteIndexEntry, IndexSeg
     }
     for (Map.Entry<String, Pair<Long, FindToken>> entry : remoteTokenTracker.getPeerReplicaAndToken().entrySet()) {
       Pair<Long, FindToken> pair = entry.getValue();
+      // Ignore this peer if the last active replication's timestamp of this peer is longer than the pre-configured days.
```

Comment:
By default, logger trace is not enabled. So we won't know if the "skip stale replica" condition is hit.
We probably can add logger.error or logger.info for the first time when we start to ignore this particular peer.

Reply:
This would not be necessary. I prefer to adding a new metric for this.